dm/CONTRIBUTING.md

```diff
@@ -73,30 +73,31 @@ questions: what changed and why. The subject line should feature the what and
 the body of the commit should describe the why.
 
 ```
-dm-master: add comment for variable declaration
+capture(ticdc): add comment for variable declaration
```

Comment:
this is dm/CONTRIBUTING.md, so the example could be related to dm

Reply:
+1

Reply:
Addressed.